doc/connectivity/bluetooth/overview.rst

```diff
@@ -93,3 +101,24 @@ Bluetooth stack.
     * Local controller support as a virtual HCI driver
 
   * Verified with multiple popular controllers
+
+* LE Audio (Auracast) in Host and Controller
```

Comment:
LE Audio and Auracast cannot be used interchangeably. 

Auracast is only the broadcast part (and the full name is Auracast broadcast audio: https://www.bluetooth.com/auracast/)

Reply:
Thanks, will remove